.env

```diff
@@ -0,0 +1,24 @@
+# Deploy directory for each service
+PROVISIONING_DEPLOY_PREFIX=/deploy/provisioning
+VERIFICATION_DEPLOY_PREFIX=/deploy/verification
+VTS_DEPLOY_PREFIX=/deploy/vts
+
+# IP address and port number for communication between VTS and the provisioning service
+VTS_PROVISIONING_COMMUNICATION_IP_ADDRESS=172.28.0.5
+VTS_PROVISIONING_COMMUNICATION_IP_ADDRESS_PORT=50051
+
+# IP address and port number for communication between VTS and the verification service
+VTS_VERIFICATION_COMMUNICATION_IP_ADDRESS=172.29.0.5
+VTS_VERIFICATION_COMMUNICATION_IP_ADDRESS_PORT=50051
+
+# Install directory for project executables
+BIN_DIR=/usr/bin
+
+# Install directory for project logs
+LOG_DIR=/var/log/veraison-logs
+
+# Install directory for project plugins
+PLUGIN_DIR=/usr/share/veraison/plugins
```

Comment:
Currently within the docker deployment because each service has its own container, so both provisioning and vts plugins have the directory name as stated above, but both vts and provisioning plugins are not in the same location

Reply:
Thanks.  And sorry, I should have removed this note before uploading the review!

Reply:
The way our plugin loading works, it necessary for them to be separate, as they implement different interfaces, and the corresponding loaders will complain if they encounter a plugin of a different type.

We should really look into unifying the two, so that there is a single plugin framework with a single loader that can accept either interface.

Reply:
> We should really look into unifying the two, so that there is a single plugin framework with a single loader that can accept either interface.

yep.  Ignorant question: do we have a tracking issue already?

Reply:
thanks!